src/function.ts

```diff
@@ -2,9 +2,16 @@ import { truncate } from './helpers.js'
 import type { Options } from './types.js'
 
 export default function inspectFunction(func: Function, options: Options) {
+  let functionType = 'Function'
+  // @ts-ignore
+  const stringTag = func[Symbol.toStringTag];
```

Comment:
just a preference but, instead of disabling the type checker, i'd cast `func` i think

```ts
type ToStringable = Function & {[Symbol.toStringTag]: Function};
```

kinda verbose but at least there's no nasty ts-ignore 😬

Reply:
Sounds good :) Weird that this isn't in the TypeScript definitions.

Reply:
you may still want to cast it btw just because functions won't have it in their normal type, rather than accepting it as a parameter i mean